### PR TITLE
docs for 1.2 release

### PR DESCRIPTION
**Pull request by sandinosanchez**

### CODE REVIEW

File: docs/config-guide.md [hunk 1]
```diff
@@ -4,115 +4,128 @@
 ### Register Organization
    
   * Open Jenkins->Management_Jobs folder.
-  * Run "RegisterOrganization" providing your SCM (GitHub) organization name as folderName
+  * Run "RegisterOrganization" providing your SCM organization name as folderName
```
Comment (vdelendik) [inline docs/config-guide.md:7]: not obligatory  requirement, so let's change to:
Run "RegisterOrganization" providing your organization name as folderName.

File: docs/config-guide.md [hunk 2]
```diff
@@ -4,115 +4,128 @@
...
-#### Setup GitHub WebHook (onPush Job/Event)
-   * Go to your GitHub repository
-   * Click "Settings" tab
-   * Click "Webhooks" menu option
-   * Click "Add webhook" button
-   * Type http://your-jenkins-domain.com/jenkins/github-webhook/ into "Payload URL" field
-   * Select application/json in "Content Type" field
-   * Tick "Send me everything." option
-   * Click "Add webhook" button
+   * Select the scm type of your repository(GitHub, Gitlab or BitBucket)
```
Comment (vdelendik) [inline docs/config-guide.md:13]: scm type selection should be moved under starting registerRepository step, doesn't it?

File: docs/config-guide.md [hunk 3]
```diff
@@ -4,115 +4,128 @@
...
-  * Create new Pull Request to your repository -> Verify that Pull request jobs are launched
+   > Note: https and ssh cloning are suported, make sure that you repository url ends up with **.git**
+
+### Setup scm webhook events (push and pull requests)
+
+ In order to enable full ci/cd workflow you need to add a webhook to your repository registered in Zebrunner, this webhook will trigger onPush and onPullRequests events on the ci component(Jenkins by default) as builds. 
+ 
+ After each Push and Pull Request related Jenkins jobs will pull your repository, run the specified test and execute the sonarqube analysys. If the webhook event is a pull request the reports from the sonarqube analysis will be decorated into the pull request for the correspondant scm system.
+ 
+ > Note: if the sonarqube server specified its not available the scan will be skipped
```
Comment (vdelendik) [inline docs/config-guide.md:26]: > Note: if the sonarqube server specified its not available the scan will be skipped
->
> Note: if the sonarqube server is not available the scan will be skipped

File: docs/config-guide.md [hunk 4]
```diff
@@ -4,115 +4,128 @@
...
+## Update Jenkins Credentials
+
+When you register a repository a credential is generated with the format of `orgName-scmType-webhook-token`, this credential needs to be updated with the correspondant scm token created in the **Setup scm webhook events (push and pull requests)** step.
+
+* Go to **Jenkins > Manage Jenkins > Manage credentials**.
+* Find your credential and click on the id.
+* Update the **Secret text** with your correspondant scm token.
+* Click save.
+
+> Note: this webhook credentials are globally used for each repository of an organisation registered in zebrunner.
```
Comment (vdelendik) [inline docs/config-guide.md:67]: organisation 
->
organization

File: docs/config-guide.md (hunk 2 above)
Comment (sandinosanchez) [reply docs/config-guide.md:13]: Its located at the second step for the Register Repository section, right after **Open your organization folder**

**Pull request merged**